test/local_sensitivity/callbacks.jl

```diff
@@ -117,8 +165,31 @@ dstuff = ForwardDiff.gradient(
 @test du02 ≈ dstuff[1:2]
 @test dp2 ≈ dstuff[3:6]
 
+cb2 = DiffEqSensitivity.track_callbacks(CallbackSet(cb),prob.tspan[1])
+sol_track = solve(prob,Tsit5(),u0=u0,p=p,callback=cb2,tstops=[5.0],abstol=1e-14,reltol=1e-14,saveat=0.1)
+cb_adj = DiffEqSensitivity.setup_reverse_callbacks(cb2,BacksolveAdjoint())
+
+adj_prob = ODEAdjointProblem(sol_track,BacksolveAdjoint(),dg!,sol_track.t,nothing,
+						 callback = cb2,
+						 abstol=1e-14,reltol=1e-14)
+adj_sol1 = solve(adj_prob, Tsit5(), tstops=sol_track.t, abstol=1e-14,reltol=1e-14)
+
+
+@test du01 ≈ -adj_sol1[1:2,end]
+@test dp1 ≈ adj_sol1[3:6,end]
+
+
+adj_prob = ODEAdjointProblem(sol_track,BacksolveAdjoint(),dg!,sol_track.t,nothing,
+						 callback = nothing,
```

Comment:
This is at the moment the odd thing.. When the reverse callback is not used, the gradients are correctly computed (because of the checkpointing..)

Reply:
oh... hahahaha what?